docs/src/enterprise/getstarted/migrate-from-fluffy.md

```diff
@@ -0,0 +1,299 @@
+    from [Docker Hub](https://hub.docker.com/u/treeverse), and a license to run lakeFS Enterprise.
+
+
+To migrate from fluffy to lakeFS Enterprise, follow the steps below:
+
+1. Sanity Test (Optional): Install a new test lakeFS Enterprise before moving your current production setup. **Make sure to include your lakeFS Enterprise license in the configuration before setup**. Test the setup → login → create repository, etc. Once everything seems to work, delete and cleanup the test setup and we will move to the migration process.
+1. Update configuration: Unlike lakeFS + Fluffy, lakeFS Enterprise uses only one configuration file. See [Configuration Changes](#configuration-changes), make sure to add the [license](#license-configuration) to the configuration.
+1. Spin down lakeFS and fluffy, and run lakeFS Enterprise!
+
+!!! warning
+    Please note that there will be a short downtime while replacing the lakeFS instances.
+
+## Configuration Changes
+
+
+### Authentication configuration
+
+Most Fluffy `auth.*` settings migrate directly to lakeFS Enterprise with the same structure. Below are the differences between the configurations.
+
+
+!!! note "SAML"
+    
+    === "lakeFS & Fluffy (old)"
+        
+        ```yaml
+        # fluffy.yaml
+        auth:  
+          logout_redirect_url: https://lakefs.company.com
+          post_login_redirect_url: https://lakefs.company.com
+          saml:
+            enabled: true 
+            sp_root_url: https://lakefs.company.com
+            sp_x509_key_path: dummy_saml_rsa.key
+            sp_x509_cert_path: dummy_saml_rsa.cert
+            sp_sign_request: true
+            sp_signature_method: http://www.w3.org/2001/04/xmldsig-more#rsa-sha256
+            idp_metadata_url: https://my.saml-provider/federationmetadata/2007-06/federationmetadata.xml
+            # idp_authn_name_id_format: "urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified"
+            external_user_id_claim_name: samName
+            # idp_metadata_file_path: 
+            # idp_skip_verify_tls_cert: true
+		```
+        ```yaml
+        # lakefs.yaml
+        auth:
+          logout_redirect_url: https://lakefs.company.com
+          cookie_auth_verification:
+            auth_source: saml
+            friendly_name_claim_name: displayName
+            persist_friendly_name: true
+            external_user_id_claim_name: samName
+            default_initial_groups:
+            - "Developers"
+        ui_config:
+          login_url: https://lakefs.company.com/sso/login-saml
+          logout_url: https://lakefs.company.com/sso/logout-saml
+          login_cookie_names:
+          - internal_auth_session
+          - saml_auth_session
+		```
+
+    
+    === "lakeFS Enterprise (new)"
+     
+        ```yaml
+        # lakefs.yaml
+        auth:
+          logout_redirect_url: https://lakefs.company.com/sso/logout-saml # optional, URL to redirect to after logout
```

Comment:
Not equal with old value and doesn't make sense this is post **logout** don't redirect to logout again